content/efcore/getting-started/samples/loading-related-data-tutorial/multiple-include-methods/Program.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var book = context.Books
+                .Include(b => b.Editions)
+                .Include(b => b.Author)
+                .Single(b => b.BookId == 1);
+
+            foreach (Edition edition in book.Editions)
+            {
+                Console.WriteLine("{0} - {1} {2}", edition.Book.Title, book.Author.FirstName, book.Author.LastName);
```

Comment:
This works fine, but doesn't really display the multiple include functionality, because we just print out the book and author like we do in the regular include example. We should print off the editions also to show that we actually do include more than the previous example.